include/circt/Dialect/RTL/RTLTypes.td

```diff
@@ -73,8 +73,8 @@ def InOutType : RTLType<"InOut"> {
 }
 
 // Type constraint that indicates that an operand/result may only be a valid
-// inout element type.
-def InOutElementType : Type<CPred<"isRTLValueType($_self)">,
+// non-directional type.
+def RTLValueType : Type<CPred<"isRTLValueType($_self)">,
```

Comment:
The RTL type system is open. This needs to accept anything not inout.

Reply:
I've opened it up for structs, but I didn't touch wire and connect.